Commands/Search.cs

```diff
@@ -0,0 +1,111 @@
+﻿
+//System
+using System;
+using System.Threading.Tasks;
+
+//Discord 
+using Discord.Commands;
+
+//Google 
+using static Google.Apis.Customsearch.v1.CseResource;
+using static Google.Apis.Customsearch.v1.CseResource.ListRequest;
+using Google.Apis.Services;
+using Google.Apis.Customsearch.v1;
+using Discord;
+
+namespace SteamDiscordBot.Commands
+{
+    public class Search : ModuleBase
+    {
+        static SafeEnum SafeSearch;
+        static int Search_Number;
+        static readonly string[] FLAGS = new string[] { "-safesearch_active", "-safesearch_off", "-num"};
+        
+        [Command("search"), Summary("Peforms google search and returns result(s)")]
+        public async Task Say(params string[] args)
+        {
+            try
+            {
+                CustomsearchService service = new CustomsearchService(new BaseClientService.Initializer { ApiKey = Program.config.GoogleApiKey });
+                SafeSearch = Program.config.GoogleSafeSearchActive ? SafeEnum.Active : SafeEnum.Off; //Default value of safesearch grabbed from settings.json
+                Search_Number = 1; //Default value number of results to return
+                string output = "";
+
+                var search = HandleSearchRequest(service, args);
+                output = FormatResultOutput(output, search);
+                if (output == "\n") { throw new ArgumentException(); } //Thrown when query input returned no result(s)
+
+                await Context.Channel.SendMessageAsync(output); 
+            }
+
+            catch (ArgumentException)
+            {
+                await Context.Channel.SendMessageAsync("No search results found");
+            }
+            catch (Exception e)
+            {
+                await Program.Instance.Log(new Discord.LogMessage(Discord.LogSeverity.Verbose, "SearchCMD", e.Message));
+                await Context.Channel.SendMessageAsync(e.Message);
+            }
+        }
+
+        static string FormatResultOutput(string output, Google.Apis.Customsearch.v1.Data.Search search)
```

Comment:
can you make these private member functions, and make the variables all non-static as well?

aka: non static